src/medLog/medLogger.cpp

```diff
@@ -112,6 +110,9 @@ void medLogger::initializeTeeStreams()
 
 void medLogger::finalizeTeeStreams()
 {
+    QObject::disconnect(this, SIGNAL(newQtMessage(QtMsgType,QString)), this, SLOT(redirectQtMessage(QtMsgType,QString)));
+    qInstallMsgHandler(NULL);
```

Comment:
The instance is deleted in ```finalisze``` so yes it is called. If it wasn't called then finalizeTeeStreams wouldn't be called anyways...
Don't regroup code that is already separated, it reduces code readability.